packages/language-support/src/parserWrapper.ts

```diff
@@ -220,7 +220,8 @@ class ParserWrapper {
   parse(query: string): EnrichedParsingResult {
     if (
       this.parsingResult !== undefined &&
-      this.parsingResult.query === query
+      this.parsingResult.query === query &&
+      process.env.BENCHMARKING !== 'true'
```

Comment:
I'm not a fan of this cause if someone had this env variable defined (unlikely but still possible) then the behaviour of the language server changes.

I'd rather probably we had a method to clean the cache and we call that after every test of the benchmark?

Reply:
I'm flexible, so if you prefer to do that on another PR, we can iterate on this

Reply:
I think it'd be nice to do in a follow up PR, just to see that the Github actions work